base/repl/LineEdit.jl

```diff
@@ -1496,8 +1498,10 @@ AnyDict(
 )
 
 const history_keymap = AnyDict(
-    "^P" => (s,o...)->(history_prev(s, mode(s).hist)),
-    "^N" => (s,o...)->(history_next(s, mode(s).hist)),
+    "^P" => (s,o...)->(edit_move_up(s) || history_prev(s, mode(s).hist)),
+    "^N" => (s,o...)->(edit_move_down(s) || history_next(s, mode(s).hist)),
```

Comment:
Aren't those bindings overwritten anyway? or is there a way to activate them easily?

Reply:
I can't really remember now, but I think that something funky happened when they weren't there.  If they don't do anything, then the similar bindings for up/down arrows should probably be removed as well.